R/compute_weighted_intensity_levels.R

```diff
@@ -0,0 +1,152 @@
+#' Compute intensity levels with weighted time series observations.
+#'
+#' @description
+#' `r lifecycle::badge("stable")`
```

Comment:
I know you include this badge because it is done throughout the package, but I would like to point out that 

1) Version number is still below 1.0.0, so by adding this badge, you are committing yourself to having this interface to the function and keeping it for a long time. I would not want to do this before a version 1.0.

2) The lifecycle package says this is not required. 
https://lifecycle.r-lib.org/articles/stages.html

"Stable

The default development stage is stable. A function is considered stable when the author is happy with its interface, doesn’t see major issues, and is happy to share it with the world. Because this stage is the default, functions will only be given a stable badge if there’s some specific need to draw attention to their status."